docs/REST API/rest-api.md

```diff
@@ -9453,7 +9459,8 @@ Add App Store (VPP) app purchased in Apple Business Manager.
 | app_store_id   | string | body | **Required.** The ID of App Store app. |
 | team_id       | integer | body | **Required**. The team ID. Adds VPP software to the specified team.  |
 | platform | string | body | The platform of the app (`darwin`, `ios`, or `ipados`). Default is `darwin`. |
-| self_service | boolean | body | Self-service software is optional and can be installed by the end user. |
+| self_service | boolean | body | Only supported for macOS apps. Specifies whether the app shows up on the **Fleet Desktop > My device** page and is available for install by the end user. |
```

Comment:
`self_service` is only supported for macOS apps. If the user sets automatic_install to true for an iOS/iPadOS app show this error:

```
Couldn’t add. Currently, self-service is only supported on macOS, Windows, and Linux. Please add the app without self_service and manually install it on the Host details page.
```

Reply:
Current error is:

```
Currently, self-service only supports macOS
```

which makes sense in the context of VPP. Sounds like we want to return an error message that covers all apps/installers, not just VPP?